### PR TITLE
actually test IPv6 in ambient

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Comment by kfaseela** [comment]
Oops

File: tests/integration/ambient/main_test.go [hunk 1]
```diff
@@ -93,7 +93,6 @@ func TestMain(m *testing.M) {
 	framework.
 		NewSuite(m).
 		RequireMinVersion(24).
-		Label(label.IPv4). // https://github.com/istio/istio/issues/41008
```
Comment (kfaseela) [inline tests/integration/ambient/main_test.go:96]: Do we have to set this based on the ipfamily of the cluster or skip completely?

Comment (howardjohn) [reply tests/integration/ambient/main_test.go:96]: The label is meant to be "Do not run this test if its in IPv6". Given ambient should run on v4 or v6 clusters, we should not have any skip

**Review by dhawton — APPROVED** [review]

**Comment by kfaseela** [comment]
Hope the test passes now! Sorry I didn't catch this before!

**Review by kfaseela — APPROVED** [review]

**Comment by kfaseela** [comment]
@bleggett so, now the L7jwt test seems to fail!

**Comment by bleggett** [comment]
We should maybe revert https://github.com/istio/test-infra/pull/5416 then for now, and chase those down.

The JWT test is probably something goofy, the `TestServiceEntryInlinedWorkloadEntry` might also be simple.

**Comment by kfaseela** [comment]
> istio/test-infra#5416

Reverted https://github.com/istio/test-infra/pull/5421

**Comment by howardjohn** [comment]
JWT failure is due to dependency on github.com, see https://github.com/istio/istio/issues/35835

**Comment by howardjohn** [comment]
ill send a fix

**Pull request merged**